Rakefile

```diff
@@ -70,6 +68,26 @@ namespace :triannon do
   desc 'run test rails console w triannon but no jetty'
   task :console => :console_no_jetty
 
+  desc 'update triannon gems and engine cart app'
+  task :update_app do
```

Comment:
I guess I didn't scroll all the way down.  If Chris's rake task works, it does look a lot simpler and cleaner.